cli/src/main/java/org/mobilitydata/gtfsvalidator/cli/Arguments.java

```diff
@@ -52,6 +55,14 @@ public class Arguments {
               + "It must be a two-letter country code (ISO 3166-1 alpha-2)")
   private String countryCode;
 
+  @Parameter(
+      names = {"--date"},
```

Comment:
I can't remember why I didn't include it initially. I think I was thinking that this is likely to be a rare argument that doesn't "need" a short form, but I agree that consistency is good. Since there's no other argument starting with d, I'll add "-d".